zeeguu_api/api/teacher_dashboard.py

```diff
@@ -215,17 +214,17 @@ def _link_teacher_cohort(user_id, cohort_id):
     return 'added teacher_cohort relationship'
 
 
-@api.route("/check_invite_code/<invite_code>", methods=["GET"])
+@api.route("/invite_code_exists/<invite_code>", methods=["GET"])
 @with_session
-def check_inv_code(invite_code):
+def inv_code_exists(invite_code):
     '''
         Checks if the inputted invite code is already in use.
 
     '''
     c = Cohort.query.filter_by(inv_code=invite_code).first()
     if c is None:
-        return jsonify(1)
-    return jsonify(0)
+        return "False"
```

Comment:
False -- is not the best answer, i'm sorry :) 
think of the implications: 

```
Q: /invite_code_exists?
A: False

```
I guess /validate_invite_code 

would be ok to have False as a result.